diff -rupN src/foam-extend-git/src/decompositionMethods/parMetisDecomp/Make/options src/foam-extend-git/src/decompositionMethods/parMetisDecomp/Make/options
--- src/foam-extend-git/src/decompositionMethods/parMetisDecomp/Make/options    2019-01-28 01:10:29.974460482 +0100
+++ src/foam-extend-git/src/decompositionMethods/parMetisDecomp/Make/options    2019-01-28 01:11:51.747793112 +0100
@@ -12,4 +12,4 @@ LIB_LIBS = \
     -L$(PARMETIS_LIB_DIR) -lmetis -lparmetis \
     -ldecompositionMethods \
     -lmetisDecomp \
-    -L$(OPENMPI_DIR)/lib -lmpi
+    -L$(OPENMPI_LIB_DIR) -lmpi
diff -rupN src/foam-extend-git/src/fvAgglomerationMethods/MGridGenGamgAgglomeration/Make/options src/foam-extend-git/src/fvAgglomerationMethods/MGridGenGamgAgglomeration/Make/options 
--- src/foam-extend-git/src/fvAgglomerationMethods/MGridGenGamgAgglomeration/Make/options   2019-01-28 01:19:52.257789089 +0100
+++ src/foam-extend-git/src/fvAgglomerationMethods/MGridGenGamgAgglomeration/Make/options   2019-01-28 03:22:08.624394281 +0100
@@ -22,11 +22,10 @@ LIB_LIBS = \
 
 EXE_INC = \
     -I$(LIB_SRC)/finiteVolume/lnInclude \
-    -I$(PARMGRIDGEN_INCLUDE_DIR)/Lib    \
-    -I$(PARMGRIDGEN_INCLUDE_DIR)/IMlib  \
+    -I$(PARMGRIDGEN_INCLUDE_DIR)    \
     $(TYPE_REAL)
 
 LIB_LIBS = \
-    -L$(PARMGRIDGEN_LIB_DIR) -lMGridGen
+    -L$(PARMGRIDGEN_LIB_DIR) -lmgrid
 
 #endif
diff -rupN src/foam-extend-git/src/dbns/Make/options src/foam-extend-git/src/dbns/Make/options 
--- src/foam-extend-git/src/dbns/Make/options       2019-01-28 01:19:52.084455758 +0100
+++ src/foam-extend-git/src/dbns/Make/options       2019-01-28 03:22:28.257727450 +0100
@@ -19,13 +19,12 @@ EXE_INC = \
     -I$(LIB_SRC)/meshTools/lnInclude \
     -I$(LIB_SRC)/thermophysicalModels/basic/lnInclude \
     -I$(LIB_SRC)/turbulenceModels/compressible/turbulenceModel \
-    -I$(PARMGRIDGEN_INCLUDE_DIR)/IMlib \
-    -I$(PARMGRIDGEN_INCLUDE_DIR)/Lib
+    -I$(PARMGRIDGEN_INCLUDE_DIR)
 
 LIB_LIBS = \
     -lfiniteVolume \
     -lmeshTools \
-    -L$(PARMGRIDGEN_LIB_DIR) -lIMlib -lMGridGen
+    -L$(PARMGRIDGEN_LIB_DIR) -lmgrid
 
 #endif
