--- src/foam-extend-git/ThirdParty/AllMake.stage5       2019-01-27 05:15:01.089724967 +0100
+++ src/foam-extend-git/ThirdParty/AllMake.stage5       2019-01-27 05:17:25.939723755 +0100
@@ -110,8 +110,11 @@ then
         # We recompile everything
         (cd ./rpmBuild/BUILD/swak4Foam-$SWAK_RELEASE_VERSION; ./Allwclean; ./Allwmake)
     else
-        echo "The source code for swak4Foam is already present under ./rpmBuild/BUILD/swak4Foam-$SWAK_RELEASE_VERSION"
-        echo "Please remove this directory if you want to refresh your installation of swak4Foam"
+#         echo "The source code for swak4Foam is already present under ./rpmBuild/BUILD/swak4Foam-$SWAK_RELEASE_VERSION"
+#         echo "Please remove this directory if you want to refresh your installation of swak4Foam"
+        (cd ./rpmBuild/BUILD/swak4Foam-$SWAK_RELEASE_VERSION; find . -name files   | xargs -n 1 sed -i.old "s/FOAM_USER/FOAM_SITE/g")
+        (cd ./rpmBuild/BUILD/swak4Foam-$SWAK_RELEASE_VERSION; find . -name options | xargs -n 1 sed -i.old "s/FOAM_USER/FOAM_SITE/g")
+        (cd ./rpmBuild/BUILD/swak4Foam-$SWAK_RELEASE_VERSION; ./Allwclean; ./Allwmake)
     fi
     echo ""
 else
--- src/foam-extend-git/ThirdParty/rpmBuild/BUILD/swak4Foam-0.4.2/Libraries/simpleSearchableSurfaces/Objects/unitCylinderSearchableSurface.C	2019-02-01 02:29:41.408802477 +0100
+++ src/foam-extend-git/ThirdParty/rpmBuild/BUILD/swak4Foam-0.4.2/Libraries/simpleSearchableSurfaces/Objects/unitCylinderSearchableSurface.C	2019-02-01 02:34:35.058800018 +0100
@@ -126,7 +126,7 @@
     point pt0=start+f[0]*dir;
     pt0.z()=0;
     if(
-        mag(topIntersect)>HUGE
+        mag(topIntersect)>GREAT
         ||
         mag(pt0)>1
     ) {
@@ -138,7 +138,7 @@
     point pt1=start+f[1]*dir;
     pt1.z()=0;
     if(
-        mag(bottomIntersect)>HUGE
+        mag(bottomIntersect)>GREAT
         ||
         magSqr(pt1)>1
     ) {
--- src/foam-extend-git/ThirdParty/rpmBuild/SPECS/mesquite-2.3.0.spec	2019-02-01 04:18:51.000000000 +0100
+++ src/foam-extend-git/ThirdParty/rpmBuild/SPECS/mesquite-2.3.0.spec	2019-02-01 04:26:33.552076912 +0100
@@ -90,6 +90,7 @@
 
 
 %build
+    patch -p0 < ../../../../../mesquite.patch
     # export WM settings in a form that GNU configure recognizes
     [ -n "$WM_CC" ]         &&  export CC="$WM_CC"
     [ -n "$WM_FC" ]         &&  export FC="$WM_FC"
