--- src/foam-extend-git/etc/bashrc	2018-06-20 01:52:27.856934810 +0200
+++ src/foam-extend-git/etc/bashrc	2018-06-20 03:22:27.347640964 +0200
@@ -47,7 +48,7 @@ export FOAM_DEV=1
 #
 # Location of FOAM installation
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-foamInstall=$HOME/$WM_PROJECT
+# foamInstall=$HOME/$WM_PROJECT
 # foamInstall=~$WM_PROJECT
 # foamInstall=/usr/local/$WM_PROJECT
 # foamInstall=/opt/$WM_PROJECT
@@ -78,7 +78,7 @@ fi
 # Location of site/user files
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 export WM_PROJECT_INST_DIR=$FOAM_INST_DIR
-export WM_PROJECT_DIR=$WM_PROJECT_INST_DIR/$WM_PROJECT-$WM_FORK-$WM_PROJECT_VERSION
+export WM_PROJECT_DIR=$WM_PROJECT_INST_DIR/$WM_PROJECT-$WM_FORK-git
 export WM_PROJECT_USER_DIR=$HOME/$WM_PROJECT/$USER-$WM_PROJECT_VERSION
 
 # Location of third-party software
--- src/foam-extend-git/bin/foamGetSystemInfo      2018-06-26 22:40:43.000000000 +0200
+++ src/foam-extend-git/bin/foamGetSystemInfo   2018-06-26 22:43:07.469667754 +0200
@@ -60,21 +60,15 @@ getNbrCores()
     # Minimal number of cores. Can be as low as 1 on virtual machines
     nbrCores=1
 
-    # First choice above all: using lstopo from the hwloc package
-    if command -v lstopo >/dev/null; then
-        nbrCores=`lstopo --of console | grep -c Core`
-    else
-        # Using some architecture specific heuristics
     case `uname -s` in
         Linux)
             # Warning: this will be wrong if Hyperthreading is enable
-            nbrCores=`grep -c processor /proc/cpuinfo`
+            nbrCores=`nproc`
         ;;
         Darwin)
             nbrCores=`sysctl -n hw.physicalcpu`
         ;;
     esac
-    fi
     echo $nbrCores
 }
