diff -rupN src/OpenFOAM-4.1/applications/utilities/mesh/generation/foamyMesh/foamyQuadMesh/CGALTriangulation2DKernel.H src/OpenFOAM-4.1/applications/utilities/mesh/generation/foamyMesh/foamyQuadMesh/CGALTriangulation2DKernel.H
--- src/OpenFOAM-4.1/applications/utilities/mesh/generation/foamyMesh/foamyQuadMesh/CGALTriangulation2DKernel.H	2016-10-16 16:11:45.000000000 +0200
+++ src/OpenFOAM-4.1/applications/utilities/mesh/generation/foamyMesh/foamyQuadMesh/CGALTriangulation2DKernel.H	2018-06-21 15:49:49.932252298 +0200
@@ -33,6 +33,7 @@ Description
 
 // * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
 
+#include "CGAL/number_utils.h"
 #include "CGAL/Delaunay_triangulation_2.h"
 
 #ifdef CGAL_INEXACT
--- src/OpenFOAM-4.1/src/OSspecific/POSIX/fileStat.C	2016-10-16 16:11:45.000000000 +0200
+++ src/OpenFOAM-4.1/src/OSspecific/POSIX/fileStat.C	2019-02-08 17:59:46.206415188 +0100
@@ -29,6 +29,7 @@ License
 
 #include <signal.h>
 #include <unistd.h>
+#include <sys/sysmacros.h>
 
 // * * * * * * * * * * * * * * * * Constructors  * * * * * * * * * * * * * * //
 
